### ISSUE TITLE
Comments on M.Sc project

### ISSUE THREAD

**Issue by Anri-Lombard** [Adamouization/Breast-Cancer-Detection-Mammogram-Deep-Learning#8]
Well done with your deep learning algorithm!

I thought of a simpler U-Net architecture for image segmentation similar to an x-ray lung cancer detector introduced by the "AI in medicine" Coursera course; although I like your ConvNet setup the softmax classifier (I hope; I can't open your paper because of its size).

Good luck with work or a Ph.D.!

**Comment by Adamouization** [comment]
Hi Anri, thanks for the comment, I appreciate it! My project colleague explored using U-Net for image segmentation, you might want to check it out.

You can try downloading the paper through my website at the very bottom if you're interested http://www.adam.jaamour.com/

**Comment by Anri-Lombard** [comment]
I have just read most of the 14,345 words, and it is overall fantastic. In fact, I thought a master's dissertation will be overwhelming (to a freshman at least), but it is explained and described so well that I could follow along. I predict this is the result of great writing!

What I learned:
(1) This paper bore down to fundamentals, even techniques like regularization (and dropout), data augmentation, and transfer learning. If you describe gradient descent it could be a full overview of machine learning :-)
(2) This paper took ethical considerations seriously; I do not know if this is common across researchers good and bad, but I respect it highly.
(3) You used imageNet for transfer learning and built your own classifier on top of it with a softmax for 3 classifications (although this is common it is the first time I see it implemented in an M.Sc. dissertation).
(4) You chose TensorFlow over Pytorch because it is older and has a larger community to help you where you get stuck. This is surprisingly clever as my naive reaction would have been "Pytorch, newer and shiny. Let's use it".

Selfish outtakes: I now understand what a master's dissertation involves and how little I know.

Lastly, I am a big fan of your website.

**Comment by Adamouization** [comment]
Thank you very much for all the comments, I really appreciate them! Ethical considerations are always very important, especially  when dealing with data like this! I do hope it is standard across all researches 😄 
PS: I also like your website, it has cool features!
